### ISSUE TITLE
WLED Staircase Usermod: unable to save number of LEDS led segments and presets

### ISSUE THREAD

**Issue by mani94321** [wled/WLED#4259]
### What happened?

I used the bin files from 
https://github.com/srg74/WLED-wemos-shield/blob/master/resources/Firmware/%40Aircoookie/Latest/WLED_0.14.4_esp32_staircase_mod.bin

and 
https://github.com/srg74/WLED-wemos-shield/blob/master/resources/Firmware/%40Aircoookie/Dev/0.15.0-b7/WLED_0.15.0-b7_esp32_staircase_mod.bin

I set the number of leds in LED preference, also created present and segments. But when WLED is restarted all setting are gone. 
tried to save segments in preset 16 but same result.
### To Reproduce Bug

Installed WLED from above sources for ESP-32. change number of LEDs and save, create presets or segments and save. Restart WLED and all segments, number of Leds are gone.

### Expected Behavior

I should be able to store the number of LEDs and segments.

### Install Method

Binary from WLED.me

### What version of WLED?

WLED 0.14.4 (build 2405180)

### Which microcontroller/board are you seeing the problem on?

ESP32

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by softhack007** [comment]
@mani94321 did you create a preset with all your segments etc, and set this in "Apply Preset at boot" in LED settings?

https://kno.wled.ge/basics/faq/#my-segments-are-gone-after-a-reboot
(the KB entry is a bit old - you may choose any preset for boot)

![image](https://github.com/user-attachments/assets/ae99ceee-a01a-4563-8397-c6dc5e3ebd4f)

**Comment by mani94321** [comment]
yes i have created segments and tried saving at different presets. I get the message encircled in red. 
![1](https://github.com/user-attachments/assets/a073681c-76c6-4864-acda-a6a7e5147605)

yes i have tried applying preset 16 at boot but after restart , all gone

.

**Comment by softhack007** [comment]
Do you use "Make a segment for each output"?
I think that there were some bugfixes in 0.15.0 related to this feature.

**Comment by mani94321** [comment]
> Do you use "Make a segment for each output"? I think that there were some bugfixes in 0.15.0 related to this feature.

no, i did not use it. Its not only the segments i cant save, the number of LEDs are also lost once WLED is restarted. 

![3](https://github.com/user-attachments/assets/9203bc06-1ec2-4d75-b43d-d2e9887624e9)
